extension/chrome/settings/modules/my_key_update.htm

```diff
@@ -16,7 +16,7 @@
   </head>
 
   <body id="settings">
-    <div id="content" class="my_key page compatibility_fix_container display_none" style="text-align: left">
+    <div id="my_key_update_container" class="my_key page display_none" style="text-align: left">
```

Comment:
I think this change can affect some styles, as we have many css rules for `body#settings > div#content`, it'll be better to leave `id="content"` and use css class here instead

Reply:
updated. thank you!